### PR TITLE
Add Nepali examples for `dos-nep-deva-latn-1997`

### PR DESCRIPTION
**Pull request by rabinshrestha**
- more examples for `dos-nep-deva-latn-1997`
- Add numeral mapping for bgnpcgn-nep-Deva-Latn-2011 as per https://geonames.nga.mil/gns/html/Romanization/ROMANIZATION%20OF%20NEPALI.pdf

### CODE REVIEW

File: maps/dos-nep-Deva-Latn-1997.yaml [hunk 1]
```diff
@@ -31,3 +45,4 @@ map:
 
     # Bisarga
     'ः': 'h'
+    'ह': 'h'
```
Comment (bilashsaha) [inline maps/dos-nep-Deva-Latn-1997.yaml]: @rabinshrestha this is inheriting the characters mapping of `bgnpcgn-nep-Deva-Latn-2011` which has a same mapping for `'ह': 'h'`. Any reason declaring this again here? I see the mapping is same here.

Comment (rabinshrestha) [reply maps/dos-nep-Deva-Latn-1997.yaml]: oh I missed this one, just removed. Thank you

**Comment by rabinshrestha** [comment]
@ronaldtse @bilashsaha I am not sure if my push caused test to fail

**Comment by bilashsaha** [comment]
@rabinshrestha I don't think so. But can you please try `rebase` you branch with `master` ?

**Comment by rabinshrestha** [comment]
@bilashsaha just rebased. Though I guess https://github.com/interscript/interscript/commit/22cf66951445155a299f2aa45c5d0626d267022a#diff-c7e2961cfdbc148f19436443ccc13108R21 is breaking test right now.

**Comment by ronaldtse** [comment]
Thanks @rabinshrestha , it should be working now. Merging.

**Pull request merged**